.envrc

```diff
@@ -0,0 +1,2 @@
+export DATABASE_URL=postgresql://fastn:fastn@172.17.0.1:5432/fastn
+use flake
```

Comment:
This is causing problem on my machine, I also have nix setup so it starts, but I do not use nix for developer setup. Can you put it in `.gitignore`?